docs/pages/web/tagdata.js

```diff
@@ -0,0 +1,247 @@
+                hideEditor
+                hideControls
+                previewHeight={200}
+              />
+            }
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use short and clear labels for easier comprehension — ideally one or two words."
+            sandpackExample={
+              <SandpackExample
+                code={doUseShortLabels}
+                name="show multiple with checkboxes"
+                hideEditor
+                previewHeight={200}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use long sentences in the text prop, since the text in TagData truncates as needed to preserve the max width."
+            sandpackExample={
+              <SandpackExample
+                code={dontUseLongLabels}
+                name="not show multiple with checkboxes"
+                hideEditor
+                hideControls
+                previewHeight={200}
+              />
+            }
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <AccessibilitySection
+        name={generatedDocGen?.displayName}
+        description="Users should be able to navigate or activate TagData using a keyboard or other input modalities. Be sure to include an `accessibilityRemoveIconLabel` and `tooltip.accessibilityLabel` for the screen reader with the `dismissable` prop and `tooltip` prop."
+      />
+
+      <MainSection
+        name="Localization"
+        description={`
+        
+      Be sure to localize \`accessibilityRemoveIconLabel\` ,\`tooltip.accessibilityLabel\`, and \`text\` properties in TagData.  
+
+      When the title of TagData reaches its max width, either intentionally or through localization, the text will be truncated with ellipses as needed to preserve the max-width. Keep this in mind when selecting wording for TagData. Note that localization can lengthen text by 20 to 30 percent.`}
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          description={`TagData is available in 3 fixed sizes.
+          - **Large** has height of 48px. Text has a fixed size of 16px. 
+          - **Medium** has height of 40px. Text has a fixed size of 14px. 
+          - **Small** has height of 32px. Text has a fixed size of 14px.
+          `}
+          title="Size"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={sizes} name="Size Variant" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="TagData should use a different fill color than the color of the surrounding UI. When used on a white background, it is recommend using a `baseColor` of white when on a gray UI background."
```

Comment:
This second sentence doesn't make any sense. I think what we're trying to convey is
"By default, TagData uses a light gray fill, which is suitable for placement on a white background. When used on a gray background, use `baseColor="white"` for a white fill."